native_client/dotnet/DeepSpeechClient/DeepSpeech.cs

```diff
@@ -83,47 +83,6 @@ public unsafe int GetModelSampleRate()
             return NativeImp.DS_GetModelSampleRate(_modelStatePP);
         }
 
-        /// <summary>
-        /// Evaluate the result code and will raise an exception if necessary.
```

Comment:
@imskr This won't compile, please see my other comment in the past commit.

Reply:
This breaks every call to `EvaluateResultCode`
Like:
https://github.com/mozilla/DeepSpeech/blob/6a135c017260761e742c5d1d8e66f478099c439d/native_client/dotnet/DeepSpeechClient/DeepSpeech.cs#L113